pkg/scannerv4/mappers/mappers_test.go

```diff
@@ -899,6 +900,222 @@ func Test_toProtoV4Contents(t *testing.T) {
+					CVE:          "CVE-7654-321",
+					Date:         "2025-01-15T00:00:00+0000",
+					EPSS:         0.04215,
+					Percentile:   0.69338,
+				},
+			},
+			want: map[string]*v4.VulnerabilityReport_Vulnerability{
+				"foo": {
+					EpssMetrics: &v4.VulnerabilityReport_Vulnerability_EPSS{
+						ModelVersion: "v2023.03.01",
+						Date:         "2025-01-15T00:00:00+0000",
+						Probability:  0.04215,
+						Percentile:   0.69338,
+					},
+					Id:          "foo",
+					Issued:      protoNow,
+					Name:        "RHSA-2021:1234",
+					Cvss:        nil,
+					CvssMetrics: nil,
+					Link:        "https://access.redhat.com/errata/RHSA-2021:1234",
+				},
+				"bar": {
+					EpssMetrics: &v4.VulnerabilityReport_Vulnerability_EPSS{
+						ModelVersion: "v2023.03.01",
+						Date:         "2025-01-15T00:00:00+0000",
+						Probability:  0.04215,
+						Percentile:   0.69338,
+					},
+					Id:          "bar",
+					Issued:      protoNow,
+					Name:        "RHSA-2021:1234",
+					Cvss:        nil,
+					CvssMetrics: nil,
+					Link:        "https://access.redhat.com/errata/RHSA-2021:1234",
+				},
+			},
+		},
+		"EPSS Missing": {
+			ccVulnerabilities: map[string]*claircore.Vulnerability{
+				"bar": {
+					ID:      "bar",
+					Name:    "Name contains CVE-5678-1234",
+					Issued:  now,
+					Updater: "unknown updater",
+				},
+			},
+			nvdVulns: map[string]map[string]*nvdschema.CVEAPIJSON20CVEItem{
+				"bar": {
+					"CVE-5678-1234": {
+						ID: "CVE-5678-1234",
+						Metrics: &nvdschema.CVEAPIJSON20CVEItemMetrics{
+							CvssMetricV31: []*nvdschema.CVEAPIJSON20CVSSV31{
+								{
+									CvssData: &nvdschema.CVSSV31{
+										Version:      "3.1",
+										VectorString: "CVSS:3.1/AV:L/AC:H/PR:L/UI:R/S:U/C:L/I:L/A:N",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			epssItems: map[string]epss.EPSSItem{
+				// No EPSS entry for CVE-5678-1234
+			},
+			want: map[string]*v4.VulnerabilityReport_Vulnerability{
+				"bar": {
+					Id:     "bar",
+					Name:   "CVE-5678-1234",
+					Issued: protoNow,
+					Cvss: &v4.VulnerabilityReport_Vulnerability_CVSS{
+						V3: &v4.VulnerabilityReport_Vulnerability_CVSS_V3{
+							Vector: "CVSS:3.1/AV:L/AC:H/PR:L/UI:R/S:U/C:L/I:L/A:N",
+						},
+						Source: v4.VulnerabilityReport_Vulnerability_CVSS_SOURCE_NVD,
+						Url:    "https://nvd.nist.gov/vuln/detail/CVE-5678-1234",
+					},
+					CvssMetrics: []*v4.VulnerabilityReport_Vulnerability_CVSS{
+						{
+							V3: &v4.VulnerabilityReport_Vulnerability_CVSS_V3{
+								Vector: "CVSS:3.1/AV:L/AC:H/PR:L/UI:R/S:U/C:L/I:L/A:N",
+							},
+							Source: v4.VulnerabilityReport_Vulnerability_CVSS_SOURCE_NVD,
+							Url:    "https://nvd.nist.gov/vuln/detail/CVE-5678-1234",
+						},
+					},
+					// No EpssMetrics because epssItems has no entry for CVE-5678-1234
+				},
+			},
+		},
+	}
+
+	ctx := context.Background()
+	for name, tt := range tests {
+		t.Run(name, func(t *testing.T) {
+			enableRedHatCVEs := "false"
```

Comment:
confirming you want this to always be false?

Reply:
For now, we can keep `enableRedHatCVEs` set to false. Once we move to the CVE-centric world, we might want to recreate some of the EPSS unit tests.